frontend/src/modules/app/OverlayScreen.tsx

```diff
@@ -79,6 +102,9 @@ const AudioAlarm = (): JSX.Element => {
   audio.loop = true;
   const [playing, setPlaying] = useState(alarmMuteStatus !== null && alarmMuteStatus.active);
 
+  /**
+   * Toggle between Play/Pause
```

Comment:
If it's possible can you give a small description in the comments here or in the file itself, why audio was handled this way? it seems like audio.play() is a promise, was wondering what was the reason behind doing it in a useEffect hook.

Reply:
Sure. I will update the following to the comments. Also we can use promise inside useEffect, so there is no specific reason apart from below added comments

```
  /**
   * Toggle between Play/Pause
   * UseEffect causes status change to Play/Pause based on `playing` state change
   * useEffect also pasues audio whenever component instance is out of context or destroyed
   * So Based on local state Audio is played or paused.
   */
```